.github/workflows/pages.yml

```
@@ -0,0 +1,230 @@
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TARGET: ${{ steps.where.outputs.target }}
+        run: |
+          set -eu
+          bash .github/scripts/publish-pages.sh "$TARGET"
+
+      - name: Say where it is
+        if: github.event_name == 'pull_request'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)